### PR TITLE
update approver/reviewer for jp

### PR DESCRIPTION
**Pull request by nasa9084**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
-- kbarnard10
-- mistyhacks
-- ryanmcginnis
-- steveperry-53
-- tengqm
-- zacharysarah
-- zparnold
+- cstoku
+- nasa9084
+- tnir
```
Comment (xiangpengzhao) [inline OWNERS]: Are these reviewers and approvers all members of Kubernetes organization? If not, I'm afraid they may not be able to lgtm/approve PRs in this repo.

**Comment by xiangpengzhao** [comment]
/assign @tnir
/assign @cstoku 
for testing

**Comment by zacharysarah** [comment]
@nasa9084 Thanks for updating these! :sparkles:

1. The `OWNERS` file looks good. 

@cstoku: Would you like to keep myself and @chenopis (as SIG Leads) as approvers? It isn't necessary, but it may be useful sometimes. If this PR is approved as is, I will no longer be able to approve any following PRs.

2. Please add [the following block](https://github.com/kubernetes/kubernetes-docs-zh/blob/master/OWNERS_ALIASES) to `OWNERS_ALIASES`:

```
aliases:
  # Emptied for localization
  # SIGs may add specific reviewers with fluency in the localization
```

3. Thanks for providing a list of `SECURITY_CONTACTS`. :+1: Please keep myself and @chenopis as security contacts; as leads for SIG Docs, we need to know about any security concerns with a localization repo.

**Comment by zacharysarah** [comment]
Thanks for the test, @xiangpengzhao. 🙇 

@tnir 👋You need to join the Kubernetes org as a [member](https://github.com/kubernetes/community/blob/master/community-membership.md#member) before you can be a reviewer/approver.

**Comment by xiangpengzhao** [comment]
:smile: 

/cc @makocchi-git @MasayaAoyama
/assign @nasa9084

This is a test for the other three folks. Those who are not org members need to join :)

**Comment by nasa9084** [comment]
I created 3 commits for @zacharysarah comment.

* keep @zacharysarah  and @chenopis (as SIG Leads) as approvers
* add the block to OWNERS_ALIASES
* Please keep @zacharysarah  and @chenopis as security contacts

**Comment by cstoku** [comment]
I agree to keep @zacharysarah and @chenopis as approvers!

Everyone except me isn't Kubernetes org member.
But these people are doing great activities at Kubernetes community.
Would it be possible for @zacharysarah and @xiangpengzhao to become their sponsor?
Or, How about becoming an approver just me at another PR first and becoming one of the their sponsor?
(if so, will @zacharysarah be another sponsor of them?)

**Comment by xiangpengzhao** [comment]
> Would it be possible for @zacharysarah and @xiangpengzhao to become their sponsor?

Sure :)

In case someone don't know the request process, below is the link:
https://github.com/kubernetes/community/blob/master/community-membership.md#member

**Comment by makocchi-ca** [comment]
@xiangpengzhao Thank you for your kindness!

**Comment by makocchi-git** [comment]
@xiangpengzhao Thank you for your kindness!

**Comment by MasayaAoyama** [comment]
@xiangpengzhao @zacharysarah @cstoku 
Thank you for your help!

**Comment by zacharysarah** [comment]
> (if so, will @zacharysarah be another sponsor of them?)

Happily! :+1:

**Comment by zacharysarah** [comment]
@nasa9084 Let's hold this PR until @makocchi-git and @MasayaAoyama are approved as members. When they are members, they can `/hold cancel`. 😄 🎉 

/hold

**Comment by nasa9084** [comment]
It's OK. I also has been requesting membership.

**Comment by tnir** [comment]
@zacharysarah Thanks. All were approved as members. kubernetes/org#32, kubernetes/org#35, kubernetes/org#36, kubernetes/org#39

/hold cancel

**Comment by cstoku** [comment]
@zacharysarah @xiangpengzhao  Thank you very much for your help!!

**Pull request merged**